src/ucs/sys/sock.c

```diff
@@ -485,3 +486,80 @@ int ucs_sockaddr_cmp(const struct sockaddr *sa1,
     }
     return result;
 }
+
+int ucs_sockaddr_is_inaddr_any(struct sockaddr *addr)
+{
+    switch (addr->sa_family) {
+    case AF_INET:
+        return UCS_SOCKET_INET_ADDR(addr).s_addr == INADDR_ANY;
+    case AF_INET6:
+        return !memcmp(&(UCS_SOCKET_INET6_ADDR(addr)), &in6addr_any,
+                       sizeof(UCS_SOCKET_INET6_ADDR(addr)));
+    default:
+        ucs_debug("Invalid address family: %d", addr->sa_family);
```

Comment:
pls start debug message with smallcase, e.g ("invalid address family: %d"
also, maybe move "return 0" after this line, instead of line 502?

Reply:
will do